voting/orbs/OrbsVoting/orbs_voting_contract.go

```diff
@@ -279,6 +279,7 @@ func getNextElectionTimeInNanos() uint64 {
  * Connections to Ethereum contracts
  */
 var ETHEREUM_TOKEN_ADDR = "0xff56Cc6b1E6dEd347aA0B7676C85AB0B3D08B0FA"
+var ETHEREUM_STAKING_ADDR = "0xff56Cc6b1E6dEd347aA0B7676C85AB0B3D08B0FA" // TODO LOCKING replace with actual contract address
```

Comment:
How many versions of the contract are there in this repo? I this looks like a duplication of `voting/orbs/Elections/` I thought the only duplication was between this repo and `/orbs-networks/orbs-network-go`